extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -36,10 +64,14 @@ public function index() {
 		$defaultModulesLimit = max( 0, self::DEFAULT_MODULES_MAX - count( $cardModulesData[ 'modules' ] ) );
 		$this->response->setValues( [
 			'heroImageUrl' => $this->getHeroImageUrl(),
-			'inviteFriendsText' => $this->msg( 'communitypage-invite-friends' )->text(),
-			'headerWelcomeMsg' => $this->msg( 'communitypage-tasks-header-welcome', $wgWikiTopic ?? $wgSitename )->parse(),
-			'subheaderWelcomeMsg' => $this->msg( 'communitypage-subheader-welcome' )->text(),
-			'pageListEmptyText' => $this->msg( 'communitypage-page-list-empty' )->text(),
+			'inviteFriendsText' => $this->msg( 'communitypage-invite-friends' )
+				->text(),
+			'headerWelcomeMsg' => $this->msg( 'communitypage-tasks-header-welcome', $wgWikiTopic ?? $wgSitename )
+				->parse(),
+			'subheaderWelcomeMsg' => $this->msg( 'communitypage-subheader-welcome' )
+				->text(),
+			'pageListEmptyText' => $this->msg( 'communitypage-page-list-empty' )
+				->text(),
 			'pageTitle' => $this->msg( 'communitypage-title' )->text(),
 			'topContributors' => $this->sendRequest(
 				'CommunityPageSpecialController',
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "CommunityPageSpecialController" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)